data/countries_names.txt

```diff
@@ -49734,63 +49734,64 @@
   vi = Nicosia, Nicosia
 
 [Czech_Jihovychod_Jihomoravsky kraj Description]
-  en = Brno, Znojmo
+  en = Brno, Znojmo, Břeclav
```

Comment:
Is it English spelling? Should it be r instead of ř? And the same in other cases? How are these places named in Google Maps or Apple Maps in English locale?

Reply:
No idea. I doubt there is anything like "English spelling" for these lesser known places other than Prague, Pilsen, etc. Google Maps do not seem to have any system to it, I see e.g. Breclav, Vyskov, Pelhrimov, Kolin, but Přerov, Třebíč, Třeboň, Šumperk. (Also, České Budějovice, not Budweis, and Karlovy Vary, not Carlsbad (but Prague and Pilsen).)